src/content/cookbook/plugins/play-video.md

```diff
@@ -85,6 +85,25 @@ The `video_player` plugin can only play asset videos in iOS simulators.
 You must test network-hosted videos on physical iOS devices.
 :::
 
+### MacOS
+
+If you use network-based videos, [add the
+`com.apple.security.network.client`
+entitlement](https://docs.flutter.dev/platform-integration/macos/building#entitlements-and-the-app-sandbox)
+
+### Web
+
+> The Web platform does **not** support `dart:io`, so avoid using the `VideoPlayerController.file`
+> constructor for the plugin. Using the constructor attempts to create a
+> `VideoPlayerController.file` that will throw an `UnimplementedError`.
+
+\* Different web browsers may have different video-playback capabilities (supported formats,
+autoplay...). Check [package:video_player_web](https://pub.dev/packages/video_player_web) for more
+web-specific information.
+
+The `VideoPlayerOptions.mixWithOthers` option can't be implemented in web, at least at the moment.
+If you use this option in web it will be silently ignored.
```

Comment:
```suggestion
The `VideoPlayerOptions.mixWithOthers` option can't be implemented in web,
at least at the moment. If you use this option in web it will be silently ignored.
```